README.md

```diff
@@ -10,37 +8,76 @@ gmake world
 gmake live
 ls output/
 ```
-## Overview
+
+# Overview
 
 This repository is smartos-live, which builds a SmartOS platform image
 containing the illumos core OS components; a set of "extra" mostly
 third-party software required by illumos, by other SmartOS software, or
 for system management; and a collection of utilities comprising
 SmartOS-specific functionality found in `projects/local/`.
 
-## Contents
+# Contents
+
+- [Community](#community)
+- [Bug Reports](#bug-reports)
+- [Components of SmartOS](#components-of-smartos)
+- [Building SmartOS](#building-smartos)
+  * [Setting up a Build Environment](#setting-up-a-build-environment)
+  * [Common Tasks](#common-tasks)
+  * [Incremental Development](#incremental-development)
+  * [Proto Area](#proto-area)
+  * [Packaging and Manifests](#packaging-and-manifests)
+- [Contributing](#contributing)
+  * [Review](#review)
+  * [Upstreaming](#upstreaming)
+  * [Integration](#integration)
+  * [Testing changes](#testing-changes)
+
+# Community
+
+You can interact with the SmartOS community in a number of ways. This
+includes:
+
+* The *smartos-discuss*
+  [mailing list](https://smartos.topicbox.com/groups/smartos-discuss).
+  If you wish to send mail to the list you'll need to join, but you can view
+  and search the archives online without being a member.
+
+* In the *#smartos* IRC channel on the [Freenode IRC
+  network](https://freenode.net/).
+
+# Bug Reports
+
+If you encounter a problem, please reach out for assistance. You can
+file a [github issue](https://github.com/joyent/smartos-live/issues) for
+any problem you encounter. When filing a bug, please include the
+platform version that you're running and a description of the problem.
```

Comment:
Unrelated. If bug reports lacking info abound, we could consider and "issue template" for the repo: https://help.github.com/en/github/building-a-strong-community/about-issue-and-pull-request-templates

Reply:
Maybe later yes.